packages/dart/lib/src/network/parse_query.dart

```diff
@@ -86,6 +86,13 @@ class QueryBuilder<T extends ParseObject> {
     limiters['keys'] = concatenateArray(keys);
   }
 
+  ///Exclude specific fields from the returned query
+  ///
+  /// [String] keys not will return the columns of a result you want the data for
+  void excludeKeys(List<String> keys) {
+    limiters['excludeKeys'] = concatenateArray(keys);
+  }
+
```

Comment:
Is it like the opposite of `keysToReturn(List<String> keys)`?

Reply:
Yes. The documentation is [here](https://docs.parseplatform.org/rest/guide/#query-constraints).
Maybe you just want to remove one field from the result and this constraint makes it easier.

Reply:
Great! 👍